File: app/src/main/java/sk/company/androidboilerplate/ui/main/MainMvpView.java
package sk.company.androidboilerplate.ui.main;

import java.util.List;

import sk.company.androidboilerplate.data.model.Ribot;
import sk.company.androidboilerplate.ui.base.MvpView;

public interface MainMvpView extends MvpView {

    void showRibots(List<Ribot> ribots);

    void showRibotsEmpty();

    void showError();

}

File: app/src/main/kotlin/sk/company/androidboilerplate/ui/main/MainMvpView.kt
package sk.company.androidboilerplate.ui.main

import sk.company.androidboilerplate.data.model.Character
import sk.creanet.alexander.ui.base.MvpView

interface MainMvpView : MvpView {
    fun showLoadingProgress(show: Boolean)
    fun showCharacters(characterList: List<Character>)
}
